src/Compilers/Core/Portable/InternalUtilities/FailFast.cs

```diff
@@ -32,6 +32,7 @@ internal static void OnFatalException(Exception exception)
 #endif
 
             Environment.FailFast(exception.ToString(), exception);
+            throw null!; // to satisfy [DoesNotReturn]
```

Comment:
```suggestion
            throw ExceptionUtilities.Unreachable;
```

Reply:
This might be a good place for `#pragma warning disable`. In the future when we adopt the newer TFM we should get the `[DoesNotReturn]` attribute on `Environment.FailFast` and we can delete the pragma. @JoeRobich don't we have a mechanism to periodically search our sources for disabled warnings and turn them back on?

See [https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/coreclr/src/System.Private.CoreLib/src/System/Environment.CoreCLR.cs#L52](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/coreclr/src/System.Private.CoreLib/src/System/Environment.CoreCLR.cs#L52)

---
In reply to: [369127571](https://github.com/dotnet/roslyn/pull/41098#discussion_r369127571) [](ancestors = 369127571)

Reply:
>  @JoeRobich don't we have a mechanism to periodically search our sources for disabled warnings and turn them back on?  

@RikkiGibson If we did, I would like to know about it. Find in Files maybe?